ntbs-service/Models/Notification.cs

```diff
@@ -227,5 +233,14 @@ private string CreateTimePeriodsString(RiskFactorDetails riskFactor)
             }
             return string.Join(", ", timeStrings);
         }
+
+        private int GetAgeAtTimeOfNotification()
+        {
+            if (NotificationDate == null || PatientDetails?.Dob == null) 
+            {
+                return 0;
```

Comment:
Wouldn't it make more sense to be null in this case, and make ageatnotification a nullable int? As somone could feasibly be zero years old when they contract TB, and that would be different to this case?